### PR TITLE
added grids

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
FEAT : added grid comp
- two col grid 
- three col grid

live preview : https://gallant-hoover-b4daf6.netlify.app/

### CODE REVIEW

**Review by MarkVed17 — APPROVED** [review]

File: grids/grid.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.ct-grid {
+  display: grid;
+  /* width: 80vh; */
+  gap: 1rem;
+}
+.ct-grid-2 {
+  grid-template-columns: 1fr 1fr;
+}
+.ct-grid-3 {
+  grid-template-columns: 1fr 2fr 1fr;
```
Comment (msdeshmukh009) [inline grids/grid.css]: I think you should also make a variation for 3 equal-size columns `repeat(3,1fr)`  as that would be helpful to you.

**Review by msdeshmukh009 — APPROVED** [review]
Good to go!

**Comment by bipin7yadav** [comment]
good to go

**Review by bipin7yadav — APPROVED** [review]

File: grids/grid.css (hunk 1 above)
Comment (MiheerTamkhane) [reply grids/grid.css]: Okay I will make it tommo.

**Pull request merged**